src/controllers/TransfersController.ts

```diff
@@ -79,4 +89,115 @@ export const TransfersController = {
       void reply.status(500)
     }
   },
+
+  transferByResource: async function (
+    request: FastifyRequest<{ Params: ITransferByResource; Querystring: ITransfer }>,
+    reply: FastifyReply,
+  ): Promise<void> {
+    const {
+      params: { resourceID },
+    } = request
+    const {
+      query: { page, limit, status },
+    } = request
+
+    try {
+      const transfers = await transfersService.findTransferByResourceID({ page, limit, status, resourceID: resourceID })
+
+      void reply.status(200).send(transfers)
+    } catch (e) {
+      if (e instanceof NotFound) {
+        void reply.status(404)
+      } else {
+        logger.error(e)
```

Comment:
As there are now many different routes it would be useful to have a helpful message when logging these errors (e.g. Error occurred when fetching transfer by resource` with the error).
This goes for all routes.